FaunaDB.Client/FaunaDB.Client.csproj

```diff
@@ -87,5 +87,7 @@
     <Compile Include="Query\PageHelper.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
     <Compile Include="Types\BytesV.cs" />
+    <Compile Include="Encoding\Encoder.cs" />
```

Comment:
And what about the attributes? Currently it's in `FaunaDB.Attributes` namespace so should user want to override a field name or ignore a field they would have to import that namespace anyway